common/src/jni/main/cpp/conscrypt/native_crypto.cc

```diff
@@ -8026,7 +8062,9 @@ static jlong NativeCrypto_SSL_set_timeout(JNIEnv* env, jclass, jlong ssl_address
     }
 
     // Convert to seconds
-    long timeout = millis / 1000;
+    static const jlong INT_MAX_AS_JLONG = static_cast<jlong>(INT_MAX);
+    uint32_t timeout = static_cast<uint32_t>(std::max(0,
```

Comment:
https://support.microsoft.com/en-us/help/143208/prb-using-stl-in-windows-program-can-cause-min-max-conflicts